### PR TITLE
Issue #8

### PR DESCRIPTION
**Pull request by cam-carter**
Updated link in accordance to accessibility standards. G91

https://www.w3.org/TR/WCAG20-TECHS/G91.html

### CODE REVIEW

**Comment by cam-carter** [comment]
...I think 😄

**Comment by nihonjinrxs** [comment]
It's a good idea to link prior conversation.  For this one, that would be a link to the prior PR:
https://github.com/codeforamerica/workforwardnola/pull/149.

File: templates/careers.mustache [hunk 1]
```diff
@@ -39,7 +39,7 @@
           <h2>Get Started Now</h2>
           <p><strong>An Opportunity Center can help you get started on one of these career paths.</strong></p>
           <p>Sign up for the Opportunity Center job system and receive a personalized recommendation for next steps and a follow-up from a career advisor.</p>
-          <p><strong>Find information on certification and training programs <span class="link"><a href="https://careerpathnola.business/training-and-educucation" target="_blank">here</a></span>.</strong></p>
+          <p><strong><span class="link"><a href="https://careerpathnola.business/training-and-educucation" target="_blank">The Network for Economic Oppurtunity career training information</a></span></strong></p>
```
Comment (nihonjinrxs) [inline templates/careers.mustache]: I think your instinct for the readability of the link was correct before, and this new version seems a bit clumsy. Why not use the prior text, minus "here", and link all of it?

Comment (cam-carter) [reply templates/careers.mustache]: Sounds good!

**Pull request merged**